src/Umbraco.Infrastructure/Services/Implement/LocalizedTextService.cs

```diff
@@ -8,63 +8,86 @@
 
 namespace Umbraco.Cms.Core.Services.Implement
 {
+    /// <inheritdoc />
     public class LocalizedTextService : ILocalizedTextService
     {
         private readonly ILogger<LocalizedTextService> _logger;
         private readonly Lazy<LocalizedTextServiceFileSources> _fileSources;
-        private IDictionary<CultureInfo, IDictionary<string, IDictionary<string, string>>> _dictionarySource => _dictionarySourceLazy.Value;
-        private IDictionary<CultureInfo, IDictionary<string, string>> _noAreaDictionarySource => _noAreaDictionarySourceLazy.Value;
-        private readonly Lazy<IDictionary<CultureInfo, IDictionary<string, IDictionary<string, string>>>> _dictionarySourceLazy;
-        private readonly Lazy<IDictionary<CultureInfo, IDictionary<string, string>>> _noAreaDictionarySourceLazy;
-        private readonly char[] _splitter = new[] { '/' };
+
+        private IDictionary<CultureInfo, Lazy<IDictionary<string, IDictionary<string, string>>>> _dictionarySource =>
+            _dictionarySourceLazy.Value;
+
+        private IDictionary<CultureInfo, Lazy<IDictionary<string, string>>> _noAreaDictionarySource =>
+            _noAreaDictionarySourceLazy.Value;
+
+        private readonly Lazy<IDictionary<CultureInfo, Lazy<IDictionary<string, IDictionary<string, string>>>>>
+            _dictionarySourceLazy;
+
+        private readonly Lazy<IDictionary<CultureInfo, Lazy<IDictionary<string, string>>>> _noAreaDictionarySourceLazy;
+        private readonly char[] _splitter = new[] {'/'};
+
```

Comment:
@Zeegaan I think we should use the `CharArrays.ForwardSlash` constant instead :)
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Core/Constants-CharArrays.cs#L10-L13

Reply:
I agree, nice catch 💪